src/checkout/checkout-request-sender.ts

```diff
@@ -12,7 +12,7 @@ export default class CheckoutRequestSender {
         private _requestSender: RequestSender
     ) {}
 
-    loadCheckout(id: string, { params: { include } = {}, timeout }: RequestOptions<CheckoutParams> = {}): Promise<Response<Checkout>> {
+    loadCheckout(id: string, { params: { include } = {}, timeout }: RequestOptions<CheckoutParams> = {}): Promise<Response<any>> {
```

Comment:
Can you revert back what you did and do the following change instead? `requestSender.get<Checkout>`